### PR TITLE
pwsh: add context menu (explorer, file)

### LINKED ISSUE #1401: [Proposal] Add to pwsh 'Open here' context menus (normal & privileged access) to Explorer
Hello,

Is it possible in pwsh (powershell) to add 'Open here' context menus (normal & privileged access) to Explorer?
Thanks for your kind consideration.

### PR DESCRIPTION
**Pull request by delassiops**
Closes #1401

 [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
image

**Comment by delassiops** [comment]
end line files fixed 👍

File: bucket/pwsh.json [hunk 1]
```diff
@@ -18,6 +22,18 @@
...
+        "    if ($architecture -eq \"32bit\") { $pwshArch = \"x86\" } else {$pwshArch = \"x64\"} ",
+        "    $pwshPath = \"$dir\\pwsh.exe\".Replace('\\', '\\\\')",
+        "    'install-explorer-context', 'uninstall-explorer-context', 'install-file-context' , 'uninstall-file-context' | ForEach-Object {",
+        "    if (Test-Path \"$bucketsdir\\main\\scripts\\pwsh\\$_.reg\") {",
+        "        $content = (Get-Content \"$bucketsdir\\main\\scripts\\pwsh\\$_.reg\").Replace('$pwshPath', $pwshPath)",
+        "        $content = $content.Replace('$pwshArch', $pwshArch)",
+        "        if ($global) { $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE') }",
+        "        Set-Content \"$dir\\$_.reg\" $content -Encoding Ascii -Force",
+        "    }",
+        "}"
```
Comment (rashil2000) [inline bucket/pwsh.json:35]: This `}` is extra. Also fix the indentation here.

**Comment by rashil2000** [comment]
Why are we using 'PowerShell 7'? The version will change in future. Better to use 'PowerShell Core'.

Also see https://github.com/ScoopInstaller/Main/issues/1401#issuecomment-1047184784 for reference.

File: scripts/pwsh/install-file-context.reg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Windows Registry Editor Version 5.00
+
+[HKEY_CURRENT_USER\Software\Classes\*\shell\PowerShell7$pwshArch]
```
Comment (rashil2000) [inline scripts/pwsh/install-file-context.reg:3]: Here also, 7 -> Core

**Comment by rashil2000** [comment]
Have you tested the reg files after the change?

**Comment by delassiops** [comment]
Tested OK 👍

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**